data/channels.csv

```diff
@@ -6105,6 +6105,7 @@ ElRadio9090FM.eg,El Radio 9090 FM,,,,EG,,,c/EG,ara,,FALSE,,,,https://www.9090.fm
 ElRey.us,El Rey,,,,US,,,c/US,eng,,FALSE,,,,https://www.elreynetwork.com/,https://i.imgur.com/u8hbbij.png
 ElrodiTV.al,Elrodi TV,,,,AL,,,c/AL,sqi,,FALSE,,,,http://elrodi.tv/,https://i.imgur.com/9oSm30W.png
 ElSegundoMedia.us,El Segundo Media,,,,US,US-CA,El Segundo,s/US-CA,eng,,FALSE,,,,https://www.elsegundo.org/our-city/el-segundo-media,https://i.imgur.com/gX1yTIH.png
+ElSelloTVMadariaga.ar,El SelloTV Madariaga,,,Sello Group,AR,AR-B,General Madariaga,s/AR-B,spa,general,FALSE,,,,https://elsellotv.com.ar,https://elsellotv.com.ar/wp-content/uploads/2023/01/LogoTV23-920x613.png

```

Comment:
```suggestion
ElSelloTVMadariaga.ar,El SelloTV Madariaga,,,Sello Group,AR,AR-B,General Madariaga,s/AR-B,spa,general,FALSE,,,,https://elsellotv.com.ar,https://i.imgur.com/bHaQqlB.png
```

Logo is too large: https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md#channels